armi/cli/compareCases.py

```diff
@@ -68,14 +68,14 @@ def _addComparisonOptions(self):
             help=("Patterns for parameters to ignore in comparisons"),
         )
         parser.add_argument(
-            "--timestepMatchup",
+            "--timestepCompare",
             default=None,
             action="store",
             nargs="+",
             help=(
-                "How to line up timesteps for comparisons. Note that any timestep not listed"
-                "will not be compared. Format the key and value separated by a period. "
-                "E.g. 0.0 1.2 2.1 3.3 will swap comparisons on the 1st and 2nd timenodes"
```

Comment:
I didn't implement the old version of this because I don't know any way to keep the input arg simple for this, and it seemed complicated to design for. If we wanted to actually match up different timesteps for comparison, it would have to look something like `1.0:0.2` to compare `c1n0` ref and `c0n2` src.

Reply:
> Better is the enemy of perfect.

If you can get a feature together that works, that's better than having something broken and defunct lying around.